README.md

```diff
@@ -9,6 +9,28 @@
 
 A flexible front end to plate bases pipelines in Sequencescape
 
+## Initial Setup
+
+1. In Sequencescape, configure all Limber required data:
```

Comment:
It's actually marginally safer to run this after `bundle exec rake config:generate` as it allows limber to register its own plate purposes, which is what actually happens in production. 

I've been hoping to uncouple these steps (It comes from request types validating their plate purposes as the SSRs kept using the wrong plate purpose) but haven't had the chance. (Plan would be to store the plate purpose via name on the submission template, and then have the order perform the validation instead. This would have the added benefit of capturing the mistake earlier.)

Reply:
Ok, I'll reorder it